tests/common.go

```diff
@@ -2902,9 +2902,15 @@ func SetClusterContext(clusterConfigPath string) error {
 	// To update the rancher client for current cluster context
 	if os.Getenv("CLUSTER_PROVIDER") == drivers.ProviderRke {
 		if !strings.HasPrefix(clusterConfigPath, "/tmp/") {
-			return fmt.Errorf("clusterConfigPath cannot be an empty string for %s cluster provider", drivers.ProviderRke)
+			if clusterConfigPath == "" {
+				err = Inst().S.(*rke.Rancher).UpdateRancherClient("source-config")
+				if err != nil {
+					return fmt.Errorf("failed to update rancher client for default source cluster context with error: [%v]", err)
+				}
+			}
+			return fmt.Errorf("invalid clusterConfigPath: %s for %s cluster provider", clusterConfigPath, drivers.ProviderRke)
```

Comment:
Shouldn't this come under an else block?

Reply:
done

Reply:
done